### PR TITLE
Remove gulp-util package

### PR DESCRIPTION
**Pull request by rotous**
This PR removes the deprecated gulp-util package. As a bonus it also updates the mocha package to fix the vulnerabilities found by npm-audit.

### CODE REVIEW

**Comment by rotous** [comment]
The updated mocha uses es6 constructs that are not supported by the old node versions, so I removed those from the Travis configuration file.

File: reporters/file.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 var fs = require('fs'),
     util = require('util'),
-    gutil = require('gulp-util'),
+	PluginError = require('plugin-error'),
+	log = require('fancy-log'),
```
Comment (JustBlackBird) [inline reporters/file.js]: Please, use spaces instead of tabs :)

File: test/specs/reporters/file.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 var fs = require('fs'),
-    gutil = require('gulp-util'),
+	PluginError = require('plugin-error'),
```
Comment (JustBlackBird) [inline test/specs/reporters/file.js]: Please, use spaces instead of tabs

File: test/specs/reporters/log.js [hunk 3]
```diff
@@ -1,21 +1,21 @@
 var expect = require('chai').expect,
     File = require('vinyl'),
-    sinon = require('sinon'),
-    gutil = require('gulp-util'),
+	sinon = require('sinon'),
```
Comment (JustBlackBird) [inline test/specs/reporters/log.js]: Please, use spaces instead of tabs :)

File: reporters/file.js (hunk 1 above)
Comment (rotous) [reply reporters/file.js]: oops... 

_feels stupid_

**Comment by rotous** [comment]
Replaced all tabs with spaces

**Comment by JustBlackBird** [comment]
It took some time, but the changes are finally released as `3.0.0` :)

Thank you @rotous !

**Comment by rotous** [comment]
Was a little busy and on holiday. :-) Thanks for the review and merge!

**Pull request merged**